### PR TITLE
Fix startsWith for ie11

### PR DESCRIPTION
**Pull request by guybedford**
Seems this IE11 bug crept in on a recent PR of mine.

### CODE REVIEW

**Review by joeldenning — APPROVED** [review]

File: src/features/script-load.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ systemJSPrototype.createScript = function (url) {
   // Only add cross origin for actual cross origin
   // this is because Safari triggers for all
   // - https://bugs.webkit.org/show_bug.cgi?id=171566
-  if (!url.startsWith(baseOrigin + '/'))
+  if (url.indexOf(baseOrigin + '/'))
```
Comment (shirotech) [inline src/features/script-load.js:23]: @guybedford Shouldn't we be checking for -1 or > 0?   returning 0 (falsy) is in fact true for the check.

**Comment by guybedford** [comment]
-1 and > 0 are both truthy! So it works.

**Comment by shirotech** [comment]
> -1 and > 0 are both truthy! So it works.

Yes, but you're not doing that...

```if (url.indexOf(baseOrigin + '/'))```

Will return false, when it returns 0 which should be true.

**Comment by guybedford** [comment]
The goal of the codepath is only to add cross origin when the URL does not
start with baseOrigin. I’d when baseOrigin is not in the string (-1) or a
random subset (> 0), both of which are the definition for the truthy values
matched.

**Pull request merged**